azure-devops/azext_devops/dev/repos/repository.py

```diff
@@ -95,12 +93,10 @@ def list_repos(devops_organization=None, project=None, detect=None):
         raise CLIError(ex)
 
 
-def show_repo(repo_id=None, name=None, devops_organization=None, project=None, detect=None, open_browser=False):
+def show_repo(repo, devops_organization=None, project=None, detect=None, open_browser=False):
     """Get the details of a Git repository.
-    :param repo_id: ID of the repository. Required if --name is not specified.
-    :type repo_id: str
-    :param name: Name of the repository. Ignored if --id is specified.
-    :type name: str
+    :param repo: ID or name of the repository. Required if --name is not specified.
```

Comment:
--name was removed. Pls update the help text.

Reply:
done